### ISSUE TITLE
ENH: Add more solar irradiance proxies

### ISSUE THREAD

**Issue by jklenzing** [pysat/pysatSpaceWeather#43]
# Description
Numerous solar proxies are available through https://lasp.colorado.edu/lisird/.  Current, pysatSpaceWeather accesses some of the F10.7 products here.  These routines could be generalized to support other indices such as Mg II core-to-wing, FISM daily bins, etc.

## Potential impact

- Is the feature related to an existing problem?
  - F10.7 does not capture short-term variability of solar inputs into the ionosphere
- How critical is this feature to your workflow?
  -  This would enable one-stop access of multiple solar irradiance proxies
- How wide of an impact to you anticipate this enhancement having?
  - Potential use in driving ITM models with multiple inputs
- Would this break any existing functionality?
  - This would be adding an instrument.  Generalizing the routines may have impact depending on implementation
  
## Potential solution(s)
- Additional instruments for lisird, with methods file for access methods

# Alternatives
A clear description of any alternative solutions or features you've considered.
- The exact instrument structure requires discussion, as the existing F10.7 will probably remain as is

# Additional context
Add any other context or screenshots about the feature request here, potentially
including your operational configuration.

**Comment by aburrell** [comment]
There are currently about 112 data sets available.  Which ones do you want most?

**Comment by jklenzing** [comment]
FISM is probably the top priority for current projects.

**Comment by aburrell** [comment]
There are many FISM options, make a checklist with the ones you want and I'll try to get at least one in for the next release.

**Comment by jklenzing** [comment]
checklist added

**Comment by aburrell** [comment]
Thought: FISM2, as a model, may be more appropriately added to pysatModels.

**Comment by jklenzing** [comment]
> Thought: FISM2, as a model, may be more appropriately added to pysatModels.

I like this idea, but note that it would likely make pysatSpaceWeather a dependency there since we use the lisird interface

**Comment by aburrell** [comment]
Not necessarily, we could instead add the lisird interface to each module individually, format it for an individual instrument (as is currently done), add it to pysat, or create a pysatLISIRD interface.  So many possibilities! 🦑
